agentarchives/archivesspace/client.py

```diff
@@ -827,7 +827,7 @@ def add_child(self, parent, title="", level="", start_date="", end_date="", date
 
         if note and note.get('content'):
```

Comment:
If this check passes, shouldn't `'content'` exist below? Otherwise `note.get('content')` returns None and the `and` fails.  Could also be `'content' in note`, and just `content = note['content']` below.